applications/callflow/src/cf_exe.erl

```diff
@@ -68,7 +69,7 @@
                ,flows = [] :: kz_json:objects()
                ,cf_module_pid :: kz_term:api_pid_ref()
                ,cf_module_old_pid :: kz_term:api_pid_ref()
-               ,status = <<"sane">> :: kz_term:ne_binary()
+               ,status = <<"init">> :: kz_term:ne_binary()
```

Comment:
Is there a specific reason to keep this a binary? could we change it to an atom (then dialyzer can be more helpful in checking that `status` is a known value).